libkbfs/block_retrieval_queue.go

```diff
@@ -16,16 +16,31 @@ import (
 
 const (
 	defaultBlockRetrievalWorkerQueueSize int = 100
+	defaultNumPrefetchWorkers            int = 1
 	testBlockRetrievalWorkerQueueSize    int = 5
 	defaultOnDemandRequestPriority       int = 100
 )
 
-type blockRetrievalConfig interface {
+type blockRetrievalParentConfig interface {
```

Comment:
The word "parent" is throwing me off a bit for some reason.  Maybe "partial" or "sub" instead?